src/Codata/Musical/Conat/Base.agda

```diff
@@ -0,0 +1,40 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Coinductive "natural" numbers: base type and operations
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}
+
+module Codata.Musical.Conat.Base where
+
+open import Codata.Musical.Notation
+open import Data.Nat.Base using (ℕ; zero; suc)
+open import Function.Base using (_∋_)
+
+------------------------------------------------------------------------
+-- The type
+
+data Coℕ : Set where
+  zero : Coℕ
+  suc  : (n : ∞ Coℕ) → Coℕ
+
+------------------------------------------------------------------------
+-- Some operations
+
+pred : Coℕ → Coℕ
+pred zero    = zero
+pred (suc n) = ♭ n
+
+fromℕ : ℕ → Coℕ
+fromℕ zero    = zero
+fromℕ (suc n) = suc (♯ fromℕ n)
+
+∞ℕ : Coℕ
```

Comment:
Very nitpicky but strictly speaking this isn't an operation. Can it go under it's own `Constants` section?